template.tpl

```diff
@@ -62,6 +62,20 @@ ___TEMPLATE_PARAMETERS___
     "simpleValueType": true,
     "help": "When this checkbox is enabled the tracker will write verbose logs to the browser's developer console.\nWe suggest turning this on only while testing your tracking script in the GTM preview mode. Disabling it reduces the size of the Leadfeeder tracking script and is a suggested option for the production websites.\n",
     "alwaysInSummary": true
+  },
+  {
+    "type": "TEXT",
+    "name": "tracking_cookie_duration_days",
+    "displayName": "[Applicable if Leadfeeder Cookie enabled] The amount of days until the Leadfeeder Cookie expires",
+    "simpleValueType": true,
+    "defaultValue": 730,
+    "valueValidators": [
+      {
+        "type": "POSITIVE_NUMBER"
+      }
+    ],
+    "help": "Defines for how many days users are tracked via Leadfeeder tracking cookie.\nThis setting is ignored in case Leadfeeder Cookie is disabled in Leadfeeder app settings.\n",
```

Comment:
```suggestion
    "help": "Defines for how many days between visits visitors are considered the same user. \nThis setting is ignored in case Leadfeeder Cookie is disabled in Leadfeeder app settings.\n",
```